From 05bd343bca468aa9c1abdec6daac235327a8048a Mon Sep 17 00:00:00 2001
From: Kedareswara rao Appana <appana.durga.rao@xilinx.com>
Date: Wed, 13 Jul 2016 14:50:34 +0530
Subject: [PATCH 049/827] dma: xilinx: Update test clients depends config option

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Updated the test client config option with the
latest driver config.

Signed-off-by: Kedareswara rao Appana <appanad@xilinx.com>
(cherry picked from commit 6300822cdc45d57bb17b75a193b909f853094507)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/dma/xilinx/Kconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/dma/xilinx/Kconfig b/drivers/dma/xilinx/Kconfig
index 0480f0e..c61796c 100644
--- a/drivers/dma/xilinx/Kconfig
+++ b/drivers/dma/xilinx/Kconfig
@@ -13,21 +13,21 @@ if XILINX_DMA_ENGINES
 
 config XILINX_DMATEST
 	tristate "DMA Test client for AXI DMA"
-	depends on XILINX_AXIDMA
+	depends on XILINX_DMA
 	help
 	  Simple DMA test client. Say N unless you're debugging a
 	  DMA Device driver.
 
 config XILINX_VDMATEST
 	tristate "DMA Test client for VDMA"
-	depends on XILINX_VDMA
+	depends on XILINX_DMA
 	help
 	  Simple DMA test client. Say N unless you're debugging a
 	  DMA Device driver.
 
 config XILINX_CDMATEST
 	tristate "DMA Test client for CDMA"
-	depends on XILINX_AXICDMA
+	depends on XILINX_DMA
 	help
 	  Simple DMA test client. Say N unless you're debugging a
 	  DMA Device driver.
-- 
2.9.3

